### ISSUE TITLE
coordinates GT scaling issue: is normalisation required?

### ISSUE THREAD

**Issue by StormArcher** [pakheiyeung/PlaneInVol#6]
Dear Author,
    We have sampled the central, lower-left and lower-right points of our data slices and found that their values are basically in the range of 0–160.
We noticed that your algorithm’s final output passes through an nn.Linear layer.
Would this require us to multiply the predicted values by 30 or divide the ground-truth by 30?

Questions:

    1 We did not perform such scaling (magnification or reduction) before, which led to poor training results. We would like to hear your advice on whether doing so would be better.
    2 We would also like to confirm whether your code normalises the ground-truth, as we noticed that the code does not seem to do so.
    3 Are your data coordinates also in the range of 0–160 or 0–10? We would like to know the value range of your slice key points, or the scale of your volumetric data.

Thank you very much for your outstanding contribution. We have carefully studied your code and hope to receive guidance from a authoritative expert.

**Comment by jayroopramesh** [comment]
Hello! Can we recommend using the latest version of this repository: https://github.com/jayroopramesh/QAERTS. Perhaps this will also help your queries!

As for those questions, in general, I suggest:
0) Why 30? If the H/W of your volume is (3,3) then (H/2) should be done.
1) Scaling does make a difference.
2) No, ground truth stays as sliced, because the entire volume (voxel-vise) is normalized at  img_vol = img_vol/img_vol.max(). 
3) It is 0-160!
